src/frontend/components/Chat/StudentChat/index.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import { wrapInIntlProvider } from '../../../utils/tests/intl';
+import { useMessagesState } from 'data/stores/useMessagesStore';
+import { DateTime } from 'luxon';
+import { StudentChat } from '.';
+import { converse } from '../../../utils/window';
+import userEvent from '@testing-library/user-event';
+
```

Comment:
ditto path please

Reply:
fix in [30417701d10c51da927b5c824ee5a16a5259c4a5](https://github.com/openfun/marsha/pull/1247/commits/30417701d10c51da927b5c824ee5a16a5259c4a5)